src/_pluggy/__init__.py

```diff
@@ -0,0 +1,26 @@
+#
```

Comment:
I was going to name it `pluggy` but that's the name of the library so I went with the PEP8 naming convention. Anyway, your suggestions make more sense.